spec/unit/blockscore/question_set_spec.rb

```diff
@@ -38,17 +38,30 @@ module BlockScore
       let(:incorrect_answers) { QuestionSetHelper.incorrect_answers(question_set.questions) }
 
       context 'correct answers' do
-        before { question_set.score(correct_answers) }
-
-        its(:score) { should eq 100.0 }
-        it { expect(question_set.score(correct_answers)).to eq 100.0 }
+        it { expect(question_set.score(correct_answers)).to be 100.0 }
```

Comment:
Instead of:

`it { expect(question_set.score(correct_answers)).to be 100.0 }`

I meant to use:

`it { expect(question_set.score(correct_answers)).to be(100.0) }`